domain-services/fiscal-event-post-processor/src/main/java/org/egov/consumer/FiscalEventPostConsumer.java

```diff
@@ -22,17 +22,17 @@
 
     @Autowired
     private ObjectMapper mapper;
-
-    @KafkaListener(topics = { "${fiscal.event.kafka.push.topic}"})
+    /**/
+    @KafkaListener(topics = {"${fiscal.event.kafka.push.topic}"})
     public void listen(final HashMap<String, Object> record, @Header(KafkaHeaders.RECEIVED_TOPIC) String topic) {
 
         try {
-            log.info("Received fiscal event post request " + record);
+            log.info("Received fiscal event post request : " + record);
```

Comment:
Remember to remove this before promoting the build to QA. 
We should not log each and every record.